server/src/services/listings.ts

```diff
@@ -35,4 +42,17 @@ const getAllListingsWithIds = async (
 const getListing = async (listingId: number): Promise<ListingResponse> =>
   listingStorage.getListing(listingId);
 
+const updateListingSuccess = async (listingId: number) => {
+  const listing = await listingStorage.getListing(listingId);
+  const isSuccessful = listing.numCommits >= listing.minCommits;
+  const affectedCommits = await commitStorage.getAllCommits([
```

Comment:
do you need to check status of commit?

Reply:
@ifhuang By right it should all be 'ongoing' at this point. Unless they were manually changed from the Datastore.